modules/sara_r4/cellular_r4_api.c

```diff
@@ -564,16 +629,106 @@ CellularError_t Cellular_SocketRecv( CellularHandle_t cellularHandle,
     }
     else
     {
-        sessionId = _Cellular_GetSessionId( pContext, socketHandle->socketId );
+        cellularStatus = _Cellular_SocketRecv( cellularHandle,
+                                               socketHandle,
+                                               pBuffer,
+                                               bufferLength,
+                                               pReceivedDataLength,
+                                               NULL,
+                                               false );
+    }
 
-        if( sessionId == INVALID_SESSION_ID )
+    return cellularStatus;
+}
+
+/*-----------------------------------------------------------*/
+
+static CellularPktStatus_t socketSendDataPrefix( void * pCallbackContext,
```

Comment:
This function hasn't been changed. Suggest to move this function to hide the difference to reduce review effort.

Reply:
Done